src/microbe_stage/systems/MicrobeMovementSystem.cs

```diff
@@ -83,12 +83,10 @@ protected override void Update(float delta, in Entity entity)
         var lookVector = control.LookAtPoint - position.Position;
         lookVector.Y = 0;
 
-        var length = lookVector.Length();
-
-        if (length > MathUtils.EPSILON)
+        if (lookVector.Length() > MathUtils.EPSILON)
         {
             // Normalize vector when it has a length
-            lookVector /= length;
+            lookVector = lookVector.Normalized();
```

Comment:
Why is the length now calculated twice? (Normalization of a vector must calculate the length).